src/data/list/count.lean

```diff
@@ -50,6 +53,9 @@ by simp only [countp_eq_length_filter, filter_append, length_append]
 lemma countp_pos {l} : 0 < countp p l ↔ ∃ a ∈ l, p a :=
 by simp only [countp_eq_length_filter, length_pos_iff_exists_mem, mem_filter, exists_prop]
 
+theorem countp_eq_zero {l} : countp p l = 0 ↔ ∀ a ∈ l, ¬ p a :=
```

Comment:
Do you want to add the `count_eq_zero` version of this too?

Reply:
Thanks, added.